Update validation method of protocols communication test (#1101)
wazuh/wazuh-qa: docs/tests/integration/test_remoted/test_agent_communication/test_protocols_communication.md

- **TCP,UDP and port 56000**
- **UDP,TCP and port 56000**

## Comments

> An important aspect to take into account is the time in which remoted makes a reload of the `client.keys` info.

Reviewer: ```suggestion
An important aspect to take into account is the time needed by wazuh-remoted to reload the `client.keys`.
```

Author: Done in https://github.com/wazuh/wazuh-qa/pull/1101/commits/fd00b88abcd5a4dcf3af786dc0a7f86fb3d3a662
